### PR TITLE
Mobile Popup Menu Functionality

### PR DESCRIPTION
**Pull request by marvin-nyalik**
In this branch the following changes were effected:
1. Addition of the main.js file for JavaScript Code
2. Addition of functionality to the menu button on mobile screen
3. Addition of links to the various sections of the page in the popup menu

Kindly let me know of any required changes

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -4,6 +4,67 @@
...
+  background: #6070ff;
+  display: none;
+  flex-direction: column;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  mix-blend-mode: multiply;
+}
+
```
Comment (rotimiazeez) [inline styles.css:20]: Please ensure your mobile menu style matches the given Figma design to create a better UI design. 👍🏽
Figma design | Your Design
Screenshot from 2023-05-16 11-58-56 | Screenshot from 2023-05-16 11-52-56.

**Review by rotimiazeez — COMMENTED** [review]
# Hi @marvin-nyalik, 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

This is supposed to be a pair-programming activity that should be done with your partner. Please ensure your partner contributes significantly to the milestone (at least make the changes pointed out here) or you'll have to re-do this milestone in a new PR 🙏🏽 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: styles.css (hunk 1 above)
Comment (marvin-nyalik) [reply styles.css:20]: I have made the required changes as requested above with my coding partner as can been seen [in this commit](https://github.com/marvin-nyalik/NyalikPortfolio/commit/a3645ee2dd8662978834ed0a9bd91ea84f895c36) and [here as well](https://github.com/marvin-nyalik/NyalikPortfolio/commit/4653d27ffe9b6b77ed3aa8353578220dc37aaac1).

Let me know of any more required changes😊...

Comment (KDlamini) [reply styles.css]: Well resolved ✔️ 

image

**Review by KDlamini — APPROVED** [review]
### Hi team @marvin-nyalik,  @sheldonwafula 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the requirements for a responsive mobile menu. 🎊 🥳 
Fantastic job nailing it on your first review.👏🏾  
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Good PR title and description
✔️ Mobile hamburger menu button is only visible on the mobile view
✔️ Mobile menu appears and closes as desired upon user interactions
✔️ The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
✔️ The JavaScript code is implemented following the [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**